File: Lab4/LW4/LW4/Data/Models/Comment.cs
namespace LW4.Data.Models;

public class Comment
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public virtual User User { get; set; } = null!;

    public int PostId { get; set; }
    public virtual Post Post { get; set; } = null!;

    public string Content { get; set; } = null!;
    public DateTime CreatedAt { get; set; } = DateTime.Now;
}

File: Lab3/MyFirstApplication/app/src/main/java/com/example/myfirstapplication/models/Comment.kt
package com.example.myfirstapplication.models

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "comments")
data class Comment(

    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,

    val userId: Int,
    val postId: Int,
    val content: String,
    val createdAt: String
)